install.sh

```diff
@@ -0,0 +1,4 @@
+# installing python 2.7 and pip for it
```

Comment:
Can we make it independent of OS? My goal is for having it on Windows as well as Linux. Also, isn't this incomplete since I'd have to `chmod +x install.sh`before running? What can we do here? Please suggest.